### PR TITLE
Don't wrap channel names

### LINKED ISSUE #716: Mobile Display Error
display error

Issue where on Samsung internet using a Samsung Galaxy S5 the channel name is not displayed correctly.

### PR DESCRIPTION
**Pull request by ItsOnlyBinary**
Supersedes #796
closes #716

### CODE REVIEW

**Comment by AllanOcelot** [comment]
Screenshot of this PR:
@prawnsalad  if you get time please could you review.

image

**Comment by ItsOnlyBinary** [comment]
rebased for easier testing

File: src/components/ContainerHeader.vue [hunk 1]
```diff
@@ -482,16 +503,20 @@ export default {
     .kiwi-header {
         margin-right: 0;
         overflow: visible;
-        height: auto;
+        height: 44px;
```
Comment (prawnsalad) [inline src/components/ContainerHeader.vue]: Any reason this height:44px should be here? This is already set on kiwi-header

File: src/components/ContainerHeader.vue [hunk 2]
```diff
@@ -396,7 +417,7 @@ export default {
 
 .kiwi-header-notjoined .u-link {
     font-weight: 600;
-    line-height: 45px;
+    line-height: 43px;
```
Comment (prawnsalad) [inline src/components/ContainerHeader.vue]: Is this 43px because its the height -1px? If so it could do with a comment explaining that

**Pull request merged**